plugins/helphub-post-types/classes/class-helphub-post-types-post-type.php

```diff
@@ -225,26 +237,25 @@ public function register_custom_column_headings( $defaults ) {
 	 * @return array           Modified array.
 	 */
 	public function updated_messages( $messages ) {
-		global $post;
+		global $post, $post_ID; /* @codingStandardsIgnoreLine */
 
-		$post_id = $post->ID;
+		$permalink = get_permalink( $post_ID ); /* @codingStandardsIgnoreLine */
```

Comment:
Same as above, this should be fine but isn't, lets revisit in a follow up audit of PHPCS ignores